docs/backends/bumble.rst

```diff
@@ -0,0 +1,116 @@
+Bumble backend
+===============
+
+This backend adds support for the |bumble| Bluetooth Controller Stack from Google.
+The backend enables support of multiple |bumble_transport| to communicate with
+a physical or virtual HCI controller.
+
+Use cases for this backend are:
+
+1. Bluetooth Functional tests without Hardware. Example of Bluetooth stacks that
+   support virtualization are |android_emulator| and |zephyr|.
+2. Connection of HCI Controllers that are not in the same radio network (virtual or physical).
+
+.. note::
+
+    To select this backend set the environmental variable ``BLEAK_BUMBLE``.
```

Comment:
The environment variable should contain the transport info.

The idea is that users should just have to set the environment variable and use any Bleak script with the Bumble backend without having to modify any code.

Reply:
I will add an env. variable to define use of bumble + transport options. In addition, I will just mention in the docs that if a specific backend must be selected then it can be changed over the `BleakClient` argument `backend` 👍